README.md

```diff
@@ -7,35 +7,41 @@ Printing since 1440.
 
 ## Editing focus
 
-> The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery. — Matt Mullenweg
+> *The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery.*
+>
+> — Matt Mullenweg
 
-One thing that sets WordPress apart from other systems is that it allows you to create as rich a post layout as you can imagine -- but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool to let you write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick it because it's what everyone else uses.
+One thing that sets WordPress apart from other systems is that it allows you to create a post layout as rich as you can imagine—but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool that can let you both write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick because it happens to be what everyone else uses.
 
-Gutenberg looks at the editor as more than a content field, revisiting a layout that has been largely unchanged for almost a decade. This allows us to holistically design a modern editing experience and build a foundation for things to come.
+**Gutenberg is a new way forward.** It looks at the editor as more than a content field, revisiting a layout that has been largely unchanged for almost a decade. This project allows us to holistically design a modern editing experience, and build a foundation for things to come.
 
 Here's why we're looking at the whole editing screen, as opposed to just the content field:
 
-1. The block unifies multiple interfaces. If we add that on top of the existing interface, it would _add_ complexity, as opposed to removing it.
-2. By revisiting the interface, we can modernize the writing, editing, and publishing experience, with usability and simplicity in mind, benefitting both new and casual users.
-3. When singular block interface takes center stage, it demonstrates a clear path forward for developers to create premium blocks, superior to both shortcodes and widgets.
-4. Considering the whole interface lays a solid foundation for the next focus, full site customization.
-5. Looking at the full editor screen also gives us the opportunity to drastically modernize the foundation, and take steps towards a more fluid and JavaScript-powered future that fully leverages the WordPress REST API.
+1. **The block unifies multiple interfaces.** If we add that on top of the existing interface, it would _add_ complexity, as opposed to removing it.
```

Comment:
'Introducing blocks as part of the existing editor would add complexity'

Reply:
Also noting we use 'as opposed to' here 😄

Reply:
For clarity, what are we referring to when we say, "that" in the first sentence [If we add "that"...]? I found that sentence confusing.